### ISSUE TITLE
Three issues, MacOS[Bug]

### ISSUE THREAD

**Issue by gromovalera** [Alumian/CEPGP-Retail#306]
Hello,

I am using CEPGP for over a month now on my MacOS and recently have encountered few problems that Windows users from my guild report as well. 
1. This first happened today: when players were rolling on an item I could see their rolls translated to the chat but the ML window remained empty as if everyone was passing on it. Therefore I had to distribute the item without EPGP and then add GP manually.
2. This happened many times and caused a lot of confusion: When I choose to distribute the item to one character — it ends up in the bag of another. 
3. When I press "Award to highest bidder" the addon gives the item to the person who has the biggest roll disregarding PR. That forces me to give loot manually and that takes a lot of extra time, is that a bug or just something that hasn't been implemented in the addon yet?

Thank you in advance,
Valeriya

**Comment by Alumian** [comment]
Thanks for your bug report. I'll look into issues 1 and 2 but regarding issue 3, the button actually says "Award to Highest Roller".

**Comment by gromovalera** [comment]
Thank you! I am using Classic 1.13.1
As to issue 3, am I right to assume there is no way the addon filters players by their PR and only then gives the loot to the highest roller?

**Comment by Alumian** [comment]
That's correct. The only filtering for this feature is an option to ignore people who have passed. Having said that, I'm not really sure what you mean when you say 'filter players by their PR'.

**Comment by gromovalera** [comment]
I see, thank you for a quick response. I was asking whether there is an option to "give to highest PR" instead of roller, mostly because that is the main criteria for loot distribution.

**Comment by Alumian** [comment]
There isn't a button for that but you can just click the PR header in the distribution window and it'll sort by PR.
